.github/workflows/server-test.yml

```diff
@@ -72,6 +72,16 @@ jobs:
           mysql database: 'kubevela'
           mysql root password: 'kubevelaSQL123'
 
+      - name: Set up OpenGauss
+        uses: lgj101/opengauss-action@v1.6
+        with:
+          GS_VERSION: 'latest'
```

Comment:
Please specify a version in case the "latest" version make CI unstable.

Reply:
Okay, modified